src/DssSpell.t.sol

```diff
@@ -57,12 +46,20 @@ contract DssSpellTest is DssSpellTestBase {
         // Insert new collateral tests here
     }
 
-    function testLerps() private { // make public to use
+    function testLerps() public { // make public to use
         vote(address(spell));
```

Comment:
Would be good to have sanity check here that the lerp works before casting.
Just warp a day, lerp suuccesfully, then do the vote, scheduleWaitAndCast etc..
Otherwise the lerp we test can just fail from any other reason and we won't know if it's really because it's unauthed.

Reply:
(this is non blocking for deployment)

Reply:
Beefed this up.